.github/pull_request_template.md

```diff
@@ -0,0 +1,34 @@
+## Description
```

Comment:
PR template should not be edited, please replace this with default template and add description on PR created directly.

Reply:
sure